### DISCUSSION TITLE
refactor support for change function/method signature?

### DISCUSSION THREAD

**Discussion by yangcheng** [golang/vscode-go#2235] [Discussions]
In GoLand, there is support for change method signature, very useful to add a default parameter or remove a return value when refactoring . is there any support for this in vscode ? 
It's always a 'no refactoring available ' for me

**Comment by shirady** [comment]
@yangcheng did you find a solution for "no refactoring available" using Gi in VSCode?

**Reply by yangcheng** [reply]
no, I use goland for such use cases
